components/layout/block.rs

```diff
@@ -647,6 +647,14 @@ impl BlockFlow {
         &mut self.fragment
     }
 
+    pub fn stacking_relative_position(&self, coor: CoordinateSystem) -> Rect<Au> {
```

Comment:
Blockflow is fixed. The inline one is a bit harder, I need to learn a bit more. 

Regards.